### DISCUSSION TITLE
Feedback: Getting started

### DISCUSSION THREAD

**Discussion by zonetw** [mswjs/mswjs.io#413] [Q&A]
- [Open page](https://mswjs.io/docs/getting-started)

## Feedback
If user want to run the example in their nodejs environment, the es module related code should add '.js'
which means:
```js
// src/mocks/node.js
import { setupServer } from 'msw/node'
import { handlers } from './handlers' // current 
import { handlers } from './handlers.js' // corrected

export const server = setupServer(...handlers)
```

```js
// src/index.js
import { server } from './mocks/node' // currrent
import { server } from './mocks/node.js' // corrected
 
server.listen()
 ...
```

**Comment by kettanaito** [comment]
Thanks for proposing this, @zonetw. 

Correct me if I'm wrong, but the `.js` extension is not required by ESM. It's required by Node.js, and is a part of a new module resolution algorithm that happened to ship with ESM. 

I treat this as an implied knowledge, as in, if you want to try this code in ESM, you have to know that ESM (may) require you to provide explicit module extensions. This isn't confusing to people using ESM, and it's not confusing to people still using CommonJS (which is, sadly, the majority).
